core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -364,4 +371,46 @@ public String toString() {
             ", internalMetadata=" + internalMetadata +
             '}';
    }
+
+   public static class Externalizer extends AbstractExternalizer<ReadCommittedEntry> {
```

Comment:
I have to admit I am not a fan of making ReadCommittedEntry itself serializable. Especially as all of the sub classes won't work. But I am not sure what the best thing here is to do other than a "view" object that only has the info we care about (key,value, metadata) and only send that across, which would require more changes.

Reply:
An option would be to convert the CacheEntry to an `InternalCacheEntry` just before serialization as that contains everything we would need and all the impls already support externalization afaik.